schemas/services-frontmatter.yml

```diff
@@ -1,8 +1,2 @@
 title: str()
```

Comment:
is `links` not used anymore? I thought that was still in use (though I could definitely be misremembering)

Reply:
I just checked again and not in the frontmatter, but yes in the main category, which is still in place